examples/2d/rotation.rs

```diff
@@ -1,12 +1,21 @@
 //! Demonstrates rotating entities in 2D using quaternions.
 
-use bevy::{math::ops, prelude::*};
+use bevy::{log::tracing_subscriber::field::MakeExt as _, math::ops, prelude::*};
 
 const BOUNDS: Vec2 = Vec2::new(1200.0, 640.0);
 
 fn main() {
     App::new()
-        .add_plugins(DefaultPlugins)
+        .add_plugins(DefaultPlugins.set(bevy::log::LogPlugin {
+            fmt_layer: |_| {
```

Comment:
We should change the log_layers example instead if we're not turning this on by default: https://github.com/bevyengine/bevy/blob/c392d28d30f34b7d2b69b806136b9643379f9284/examples/app/log_layers.rs#L45-L49